wagtail/admin/templates/wagtailadmin/home/ie11_warning.html

```diff
@@ -0,0 +1,5 @@
+{% load wagtailcore_tags %}
+
+<div data-ie11-warning hidden class="panel nice-padding">
+    <div class="help-block help-info">Wagtail is gradually removing support for IE11, which may affect you or your organisation. <a href="https://docs.wagtail.io/en/stable/editor_manual/browser_issues.html#ie11">Learn more about our IE11 support plans</a>.</div>
```

Comment:
This needs a {% blocktrans trimmed %}...

Reply:
@allcaps the existing "Upgrade notifications" is english-only, which is why I had this in english only as well: https://github.com/wagtail/wagtail/blob/master/wagtail/admin/templates/wagtailadmin/home/upgrade_notification.html

I’m not sure _why_ though, so yep feels like it could be nicer translated.

Reply:
Thinking about this more, it does feel to me like I overthought this (…), and it’s probably just a mistake for the upgrade notification to be english only.